website/server/libs/payments.js

```diff
@@ -109,7 +117,19 @@ api.addSubToGroupUser = async function addSubToGroupUser (member, group) {
   if (member.isSubscribed()) {
     let memberPlan = member.purchased.plan;
     let customerHasCancelledGroupPlan = memberPlan.customerId === this.constants.GROUP_PLAN_CUSTOMER_ID && !member.hasNotCancelled();
-    if (customerIdsToIgnore.indexOf(memberPlan.customerId) !== -1 && !customerHasCancelledGroupPlan) return;
+    let ignorePaymentPlan = paymentMethodsToIgnore.indexOf(memberPlan.paymentMethod) !== -1;
+    let ignoreCustomerId = customerIdsToIgnore.indexOf(memberPlan.customerId) !== -1;
+
+    if (ignorePaymentPlan) {
+      txnEmail(FLAG_REPORT_EMAILS, 'invalid-payment-cancel', [
```

Comment:
Please make template name be `admin-user-subscription-details` so that it's more generic in case we reuse it later for emailing admins about subscriptions for other reasons.